airbyte-integrations/connectors/source-gitlab/source_gitlab/schemas/project_dora_metrics_lead_time_for_changes.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "project_id": {
+        "type": ["null", "integer"]
+      },
+      "date": {
+        "type": ["null", "string"]
+      },
+      "value": {
+        "type": ["null", "number"]
+      }
+    }
+  }
+  
```

Comment:
EOF, please run `airbyte-ci format fix all`

Reply:
I wasn't able to make airbyte-ci to work, the install process is really complex. 

Edit: was able to install but it doesn't work